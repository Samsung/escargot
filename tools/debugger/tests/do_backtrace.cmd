s
backtrace
s
s
s
bt t
bt 2
bt 1 3
c
