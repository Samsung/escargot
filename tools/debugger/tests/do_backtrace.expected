Connecting to: localhost:6501
Connection created!!!
Stopped at tools/debugger/tests/do_backtrace.js:20
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace.js:34
(escargot-debugger) backtrace
tools/debugger/tests/do_backtrace.js:34:1
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace.js:31 (in h() at line:30, col:1)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace.js:27 (in g() at line:26, col:1)
(escargot-debugger) s
Stopped at tools/debugger/tests/do_backtrace.js:23 (in f() at line:22, col:1)
(escargot-debugger) bt t
Total number of frames: 4
tools/debugger/tests/do_backtrace.js:23:5 (in f)
tools/debugger/tests/do_backtrace.js:27:5 (in g)
tools/debugger/tests/do_backtrace.js:31:5 (in h)
tools/debugger/tests/do_backtrace.js:34:1
(escargot-debugger) bt 2
tools/debugger/tests/do_backtrace.js:23:5 (in f)
tools/debugger/tests/do_backtrace.js:27:5 (in g)
(escargot-debugger) bt 1 3
tools/debugger/tests/do_backtrace.js:27:5 (in g)
tools/debugger/tests/do_backtrace.js:31:5 (in h)
(escargot-debugger) c
Connection closed.
